doc/input-output-reference/src/overview/group-system-availability-managers.tex

```diff
@@ -628,6 +637,16 @@ \subsubsection{Outputs}\label{outputs-9-008}
 
 HVAC,Average,Availability Manager Hybrid Ventilation Control Status {[]}
 
+HVAC,Average,Hybrid Ventilation Control HVAC System Opration Elapsed Time {[min]}
+
+HVAC,Average,Hybrid Ventilation Control Natual Ventilation Elapsed Time {[min]}
+
+HVAC,Average,Hybrid Ventilation Operative Temperature {[C]}
```

Comment:
@mjwitte We already have zone CO2 level output variable: Zone Air CO2 Concentration [ppm]. If we add a new output variable as you suggested, it may duplicate output variables to report the same variable. Please let me know your thought.